modules/claims_api/lib/claims_api/v2/disability_compensation_pdf_mapper.rb

```diff
@@ -579,22 +581,42 @@ def get_service_pay # rubocop:disable Metrics/MethodLength
       def convert_date_string_mdy_to_object(date_string)
         return '' if date_string.blank?
 
-        date = Date.strptime(date_string, '%m-%d-%Y')
+        arr = date_string.split('-')
         {
-          month: date.mon,
-          day: date.mday,
-          year: date.year
+          month: arr[0].to_s,
+          day: arr[1].to_s,
+          year: arr[2].to_s
         }
       end
 
       def convert_date_string_my_to_object(date_string)
         return '' if date_string.blank?
 
-        date = Date.strptime(date_string, '%m-%Y')
+        arr = date_string.split('-')
         {
-          month: date.mon,
-          year: date.year
+          month: arr[0].to_s,
+          year: arr[1].to_s
+        }
+      end
+
+      def additional_identification_info
+        name = {
+          lastName: @target_veteran.last_name,
+          firstName: @target_veteran.first_name,
+          middleInitial: (@target_veteran.middle_name.presence || '')
         }
+        if @target_veteran.birth_date
+          birth_date =
+            {
+              month: @target_veteran.birth_date[4..5].to_s,
+              day: @target_veteran.birth_date[6..7].to_s,
+              year: @target_veteran.birth_date[0..3].to_s
+            }
+        end
+        @pdf_data[:data][:attributes][:identificationInformation][:name] = name
+        @pdf_data[:data][:attributes][:identificationInformation][:ssn] = @target_veteran.ssn
+        @pdf_data[:data][:attributes][:identificationInformation][:dateOfBirth] = birth_date
+        @pdf_data
```

Comment:
Since the date comes from @target_veteran, it comes in a different format from the dates in our schema.